banner/javascript/sandpack.config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
should we consider a different namespace? Sandpack is a thing and what we're doing is very related but it's not sandpack's doing

Reply:
unsure i feel like it describes it best